docs/xray/auditpip/help.go

```diff
@@ -0,0 +1,5 @@
+package auditpip
+
+const Description = "Scan a VCS project with JFrog Xray, after the sources were cloned locally."
```

Comment:
Scans a VCS project with JFrog Xray, after the sources were cloned locally.
-->
Execute an audit Pip command, using the configured Xray details.